src/Injector/InjectorInterface.php

```diff
@@ -55,4 +57,12 @@ public function inject($package, $type, IOInterface $io);
      * @return void
      */
     public function remove($package, IOInterface $io);
+
+    /**
+     * Set modules of the application.
+     *
+     * @param array $modules
+     * @return self
+     */
+    public function setApplicationModules(array $modules);
```

Comment:
We're still in the 0.X series, so we can bump to 0.4.0 for this release. I think it makes sense, based on what you're fixing about the behavior of the plugin.